si_ps_conv_fastcorrsApp/Db/kicks.template

```diff
@@ -0,0 +1,56 @@
+include "kick_curr_pairs.db"
+
+# Records for sync Kick and Current setpoints
+record(bi, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):CnctCurrentSP-Mon") {
+    field(PINI, "YES")
+    field(SCAN, "1 second")
+    field(INP,  "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):Current-SP.DOL1V" )
+    field(ZNAM, "Ext PV Not Connected")
+    field(ZSV , "MAJOR")
+    field(ONAM, "Ext PV OK")
+}
+
+record(sseq, "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):Current-SP") {
+    field(PINI, "YES")
+    field(DOL1, "SI-$(SEC)$(DIP):PS-$(ORI):Current-SP CP")
+    field(LNK1, "$(TEST)copy-SI-$(SEC)$(DIP):PS-$(ORI):Current-SP PP")
+}
+
+record(ai, "$(TEST)copy-SI-$(SEC)$(DIP):PS-$(ORI):Current-SP") {
+    field(EGU, "A")
+    field(PREC, 3)
+    field(SCAN, "Passive")
+    field(FLNK, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):KickSP-RB PP")
+}
+
+record(ao, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-SP") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DTYP, "Python Device")
+    field(OUT, "@strn2curr $(TEST) $(SEC) $(DIP) $(ORI) Current-SP")
+}
+
+record(ao, "$(TEST)out-SI-$(SEC)$(DIP):PS-$(ORI):Current-SP") {
+    field(EGU, "A")
+    field(PREC, 3)
+    field(OUT, "SI-$(SEC)$(DIP):PS-$(ORI):Current-SP CA")
+    field(SDIS, "fwd-SI-$(SEC)$(DIP):PS-$(ORI):KickSP-RB.PACT")
```

Comment:
I think it would be nice to have a comment documenting the record processing chain and its usage of only local links, so that when this record is reached, KickSP-RB will, indeed, still be processing.

proxy-Current-SP -> copy-Current-SP -> KickSP-RB -> fwd-KickSP-RB -> Kick-SP -> out-Current-SP

(this is another reason for my suggestion of using a FLNK in Kick-SP, it makes the processing chain more explicit).